dotnet-local.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+if [ -d "bin/Release/dotnet" ]; then
+    ./bin/Release/dotnet/dotnet $@
+else
```

Comment:
I'd add another `-x` check here and an `else` which would report an error if the `dotnet` binary doesn't exist or isn't executable